Code/GraphMol/MMPA/MMPA.cpp

```diff
@@ -408,13 +388,13 @@ static void addResult(std::vector<std::pair<ROMOL_SPTR, ROMOL_SPTR>>
 //=====================================================================
 static inline void appendBonds(BondVector_t &bonds,
                                const BondVector_t &matching_bonds) {
-  for (const auto &matching_bond : matching_bonds) {
-    bonds.push_back(matching_bond);
-  }
+  bonds.reserve(bonds.size() + matching_bonds.size());
+  bonds.insert(bonds.end(), matching_bonds.begin(), matching_bonds.end());
 }
 
 static inline void processCuts(
-    size_t i, size_t minCuts, size_t maxCuts, BondVector_t &bonds_selected,
+    std::size_t i, std::size_t minCuts, std::size_t maxCuts,
```

Comment:
It's definitely not wrong to do this, but we normally just use `size_t` without the `std::`.  This is the C version, not the C++ one (though they are both the same). Making this change also adds a bunch of noise to the PR.

For consistency with the rest of the code, I would really prefer to have the `std::size_t` changes reverted

Reply:
I don't feel strongly about it at all - I would have never noticed unless VSCode had flagged all of those with red squiggles. And I agree that unless we do that change on the whole RDKit codebase, it is better to leave it as it was. Actually my plan was to revert that change before committing, but then I forgot; I have reverted it now.